src/Hydrodynamics/SphSimulation.cpp

```diff
@@ -1663,6 +1729,5 @@ void SphSimulation<ndim>::SmoothParticleQuantity
  (const int Npart,
   FLOAT *values)
 {
-
```

Comment:
An empty function which is never called has no right to exist. Let's get rid of it!